src/main/java/edu/studyup/map/Lookup.java

```diff
@@ -37,13 +37,15 @@ private static Location getLocation(JSONObject location) {
 		return new Location(lat, lon, bounds);
 	}
 
+	//FIXED BUG #1: Reliance on default encoding
```

Comment:
BUG #1: Good one! Wooooow, it took me 2 days to look around on StackOverflow for possible solutions but I have not been able to. I am so glad that we have code review in this assignment.